manual/doc/extending.tex

```diff
@@ -88,4 +88,94 @@ \section{How to add a new observable}
 
 \section{How to add a new constraint}
 
-TODO
+Constraints are stored in YAML files below \directory{eos/constraints}. New constraints can be
+added to an existing file, or to an entirely new file. Each file contains key-to-value maps
+in which the top-level keys declare a new constraint.
+The values in this map are the definitions of a constraint, and are also structured as maps. Valid
+constraint definitions must at least include a \texttt{type} entry that governs the functional
+form of the associated likelihood. \EOS understands the following types of constraints:
+\begin{itemize}[labelwidth=.15\textwidth]
+    \item \texttt{Amoroso},
+    \item \texttt{Gaussian},
+    \item \texttt{MultivariateGaussian},
+    \item \texttt{MultivariateGaussian(Covariance)}.
+\end{itemize}
+
+\subsection{Type \texttt{Amoroso}}
+
+Type \texttt{Amoroso} uses a likelihood arising from the \gls{PDF} of the
+Amoroso distribution \cite{Crooks:2010}, a four-parameter distribution. This
+distribution is useful for the modelling of (sequences) of upper limits on
```

Comment:
"... of (sequences) of upper limits ..."
 maybe to
"... (of sequences) of upper limits ..."

Reply:
Rephrased.